planner:
  pose_topic: "/ground_truth"
  cmd_topic: "/gpg_velocity_controller/cmd_vel"
  map_topic: "/grid_map_filter/filtered_map"
  base_to_hip_x: 0.1881
  base_to_hip_y: 0.12675 # 0.04675
  robot_height: 0.4
  t_stance: 1.0
  k_velocity: 0.03
  gravity: 9.81
  threshold_traversability: 0.5
  

